routes/mail/mail.js

```diff
@@ -0,0 +1,56 @@
+const db = require("../../db/models");
```

Comment:
looks like this file is located in incorrect folder. Please move out from routes folder